packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -55,9 +55,10 @@ const COMMON_SPACING = 'my-4';
 
 export const DescriptionWrapper: React.FC<{
   children: string;
-}> = ({ children }) => {
+  spacing?: string;
```

Comment:
if it is a spacing it should be more typed, if it is any class name than it should be renamed